contrib/freshmaker-cli

```diff
@@ -224,6 +239,48 @@ def get_freshmaker_events_by_ids(api_url, ids):
     events = sorted(events, key=lambda k: k['id'])
     return events
 
+def get_advisories_from_errata(errata_url):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED state. Returns the
+    list of advisories that are IMPORTANT or CRITICAL.
+
+    """
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, verify=False, auth=krb_auth)
+    #https://errata.engineering.redhat.com/filter/2958.json - prod
+    #https://errata.devel.redhat.com/filter/2943.json - dev
+    #https://errata.stage.engineering.redhat.com/filter/1469.json - stage
+
+    r.raise_for_status()
+    data = r.json()
+    return [
+        advisory["id"]
+        for advisory in data
+        if advisory["security_impact"] in ("Important", "Critical")
+    ]
+
+def get_freshmaker_events_by_search_keys(api_url, search_keys):
+    print("Getting Freshmaker events")
+    events = []
+    for i in search_keys:
+        url = '%s/api/1/events/?search_key=%s' % (api_url, i)
+
+        r = requests.get(url, verify=False)
```

Comment:
It'd be nice if this timed out after 30 seconds of not hearing a response. If you don't specify a timeout and the server doesn't respond, this line of code will just hang forever.
```suggestion
        r = requests.get(url, timeout=30)
```